src/SplitIO/Component/Utils/functions.php

```diff
@@ -97,3 +95,8 @@ function getSplitEventsUrl()
             return SPLITIO_EVENTS_URL;
     }
 }
+
+function isAssoc($arr)
```

Comment:
can we change this to isAssociativeArray?

Reply:
makes sense, changing now